app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,56 @@
+class ListsController < ApplicationController
+    before_action :set_board, only: [:new, :create, :edit, :update, :destroy]
+    before_action :set_list, only: [:edit, :update, :destroy] 
+
+    def new
+        @list = @board.lists.new
+    end
+
+    def create
+        @list = @board.lists.create(create_params)
+        if @list.save
+            redirect_to board_path(@board), notice: 'List created'
+        else
+            render :new
+        end
+    end
+
+    def edit
+    end
+
+    def update
+          if @list.update_attributes
```

Comment:
? co to jest ? jakie update attribues ? skad wezmie co ma zmienic ?

Reply:
w private:
    def update_params
        params.update_attributes(name: params[:list][:name])
    end

def update
          if @list.update_params
            redirect_to boards_path(@board), notice: 'List name changed!'
          else
            render :edit
          end
    end

??

Reply:
ehhh
przeciez Ty wywołujesz te udate_attributes na @list -> musiało by to byc w modelu listy ... jako metoda instancyjna ... a nie w private w controllerze . Jak chcesz wywołać to co jest w private w kontrolerze to robisz poprostu `update_attributes` i `self` jest z defaultu instancja controllera ...